### PR TITLE
[#6945] [Platform] [Backport-2.4] Add limit to customer task API.

### PR DESCRIPTION
**Pull request by mahendranbhat**
Description:
We could make the tasks UI more functional by adding a LIMIT 2000 to our tasks DB query, so we don't end up with a spinning task UI for Platform instances that accumulate many tasks over time.
Currently it is set to 2000 in reference.conf.

Original Commit: [https://github.com/yugabyte/yugabyte-db/pull/7477] (57d835e9c13c92cf2feaaa716d5f08e6cdef9442)

Test Plan:

Updated reference.conf to 20.
Verified whether the API returning only 20 entries or not.
Also verified the working endpoint of a particular task.
Verified by creating 10 + tasks and verified pagination in UI too.

sbt test:

```info] Test com.yugabyte.yw.cloud.UniverseResourceDetailsTest.testAddPriceWithRemovingOneNode started
[info] Test run finished: 0 failed, 0 ignored, 7 total, 1.009s
[info] Passed: Total 1131, Failed 0, Errors 0, Passed 1127, Skipped 4
[success] Total time: 439 s, completed 1 Jun, 2021 10:45:19 PM```

### CODE REVIEW

**Review by sb-yb — APPROVED** [review]
Thanks!

File: managed/src/test/java/com/yugabyte/yw/controllers/CustomerTaskControllerTest.java [hunk 1]
```diff
@@ -50,12 +54,18 @@
...
+import org.mockito.junit.MockitoJUnitRunner;
+import org.mockito.InjectMocks;
 
+@RunWith(MockitoJUnitRunner.class)
 public class CustomerTaskControllerTest extends FakeDBApplication {
   private Customer customer;
   private Users user;
   private Universe universe;
 
+  @InjectMocks private CustomerTaskController controller;
```
Comment (sb-yb) [inline managed/src/test/java/com/yugabyte/yw/controllers/CustomerTaskControllerTest.java:67]: I sthis still needed? For commissioner status mocking?

**Review by sb-yb — APPROVED** [review]
1 question.

File: managed/src/test/java/com/yugabyte/yw/controllers/CustomerTaskControllerTest.java (hunk 1 above)
Comment (mahendranbhat) [reply managed/src/test/java/com/yugabyte/yw/controllers/CustomerTaskControllerTest.java]: we do not need this, removed.

**Pull request merged**